apps/src/templates/teacherHomepage/UiTip.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+const UiTip = React.createClass({
+  propTypes: {
+    index: React.PropTypes.number,
+    position: React.PropTypes.object,
+    text: React.PropTypes.string,
+    closeClicked: React.PropTypes.func.isRequired
+  },
+
+  closeClicked(index) {
+    this.props.closeClicked(index);
+  },
+
+  render() {
+    const { index, position, text } = this.props;
+
+    return (
+      <div>
+        <div
+          className="arrow_box_up"
+          onClick={this.closeClicked.bind(this, index)}
```

Comment:
shouldnt need to bind closeClicked (since this is using `createClass` which autobinds class methods)